README.md

```diff
@@ -26,6 +26,28 @@ If you need support for other Google APIs, please check out the [Google APIs Cli
 $ composer require google/cloud
 ```
 
+## Caching Access Tokens
+
+It is possible to cache access tokens by passing a [PSR-6](http://www.php-fig.org/psr/psr-6/) caching implementation in to the desired client.
+
+The following example takes advantage of [Stash](https://github.com/tedious/Stash).
+
+```php
+<?php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+use Stash\Driver\FileSystem;
+use Stash\Pool;
+
+$driver = new FileSystem([]);
```

Comment:
Just one small concern. Will the file cache here be stored securely? If it contains access tokens, no one else should be able to read that file.